### ISSUE TITLE
Plugin is broken in 2016.2.4 (Kotlin 1.0.4)

### ISSUE THREAD

**Issue by raniejade** [raniejade/spek-idea-plugin#10]
```
9/24/16
12:40 PM	NoSuchMethodError: org.jetbrains.kotlin.asJava.LightClassUtilsKt.toLightClass(Lorg/jetbrains/kotlin/psi/KtClassOrObject;)Lorg/jetbrains/kotlin/asJava/KtLightClass;

12:41 PM	NoSuchMethodError: org.jetbrains.kotlin.asJava.LightClassUtilsKt.toLightClass(Lorg/jetbrains/kotlin/psi/KtClassOrObject;)Lorg/jetbrains/kotlin/asJava/KtLightClass;

12:41 PM	NoSuchMethodError: org.jetbrains.kotlin.asJava.LightClassUtilsKt.toLightClass(Lorg/jetbrains/kotlin/psi/KtClassOrObject;)Lorg/jetbrains/kotlin/asJava/KtLightClass;

12:41 PM	NoSuchMethodError: org.jetbrains.kotlin.asJava.LightClassUtilsKt.toLightClass(Lorg/jetbrains/kotlin/psi/KtClassOrObject;)Lorg/jetbrains/kotlin/asJava/KtLightClass;

12:41 PM	NoSuchMethodError: org.jetbrains.kotlin.asJava.LightClassUtilsKt.toLightClass(Lorg/jetbrains/kotlin/psi/KtClassOrObject;)Lorg/jetbrains/kotlin/asJava/KtLightClass;

12:41 PM	NoSuchMethodError: org.jetbrains.kotlin.asJava.LightClassUtilsKt.toLightClass(Lorg/jetbrains/kotlin/psi/KtClassOrObject;)Lorg/jetbrains/kotlin/asJava/KtLightClass;

12:41 PM	NoSuchMethodError: org.jetbrains.kotlin.asJava.LightClassUtilsKt.toLightClass(Lorg/jetbrains/kotlin/psi/KtClassOrObject;)Lorg/jetbrains/kotlin/asJava/KtLightClass;

12:41 PM	NoSuchMethodError: org.jetbrains.kotlin.asJava.LightClassUtilsKt.toLightClass(Lorg/jetbrains/kotlin/psi/KtClassOrObject;)Lorg/jetbrains/kotlin/asJava/KtLightClass;

12:41 PM	NoSuchMethodError: org.jetbrains.kotlin.asJava.LightClassUtilsKt.toLightClass(Lorg/jetbrains/kotlin/psi/KtClassOrObject;)Lorg/jetbrains/kotlin/asJava/KtLightClass;

12:41 PM	NoSuchMethodError: org.jetbrains.kotlin.asJava.LightClassUtilsKt.toLightClass(Lorg/jetbrains/kotlin/psi/KtClassOrObject;)Lorg/jetbrains/kotlin/asJava/KtLightClass;

12:41 PM	NoSuchMethodError: org.jetbrains.kotlin.asJava.LightClassUtilsKt.toLightClass(Lorg/jetbrains/kotlin/psi/KtClassOrObject;)Lorg/jetbrains/kotlin/asJava/KtLightClass;

12:41 PM	NoSuchMethodError: org.jetbrains.kotlin.asJava.LightClassUtilsKt.toLightClass(Lorg/jetbrains/kotlin/psi/KtClassOrObject;)Lorg/jetbrains/kotlin/asJava/KtLightClass;

12:41 PM	NoSuchMethodError: org.jetbrains.kotlin.asJava.LightClassUtilsKt.toLightClass(Lorg/jetbrains/kotlin/psi/KtClassOrObject;)Lorg/jetbrains/kotlin/asJava/KtLightClass;

12:41 PM	NoSuchMethodError: org.jetbrains.kotlin.asJava.LightClassUtilsKt.toLightClass(Lorg/jetbrains/kotlin/psi/KtClassOrObject;)Lorg/jetbrains/kotlin/asJava/KtLightClass;

12:41 PM	NoSuchMethodError: org.jetbrains.kotlin.asJava.LightClassUtilsKt.toLightClass(Lorg/jetbrains/kotlin/psi/KtClassOrObject;)Lorg/jetbrains/kotlin/asJava/KtLightClass;

12:41 PM	NoSuchMethodError: org.jetbrains.kotlin.asJava.LightClassUtilsKt.toLightClass(Lorg/jetbrains/kotlin/psi/KtClassOrObject;)Lorg/jetbrains/kotlin/asJava/KtLightClass;

12:42 PM	NoSuchMethodError: org.jetbrains.kotlin.asJava.LightClassUtilsKt.toLightClass(Lorg/jetbrains/kotlin/psi/KtClassOrObject;)Lorg/jetbrains/kotlin/asJava/KtLightClass;

12:42 PM	NoSuchMethodError: org.jetbrains.kotlin.asJava.LightClassUtilsKt.toLightClass(Lorg/jetbrains/kotlin/psi/KtClassOrObject;)Lorg/jetbrains/kotlin/asJava/KtLightClass;

12:42 PM	NoSuchMethodError: org.jetbrains.kotlin.asJava.LightClassUtilsKt.toLightClass(Lorg/jetbrains/kotlin/psi/KtClassOrObject;)Lorg/jetbrains/kotlin/asJava/KtLightClass;

12:42 PM	NoSuchMethodError: org.jetbrains.kotlin.asJava.LightClassUtilsKt.toLightClass(Lorg/jetbrains/kotlin/psi/KtClassOrObject;)Lorg/jetbrains/kotlin/asJava/KtLightClass;

12:42 PM	NoSuchMethodError: org.jetbrains.kotlin.asJava.LightClassUtilsKt.toLightClass(Lorg/jetbrains/kotlin/psi/KtClassOrObject;)Lorg/jetbrains/kotlin/asJava/KtLightClass;

12:42 PM	NoSuchMethodError: org.jetbrains.kotlin.asJava.LightClassUtilsKt.toLightClass(Lorg/jetbrains/kotlin/psi/KtClassOrObject;)Lorg/jetbrains/kotlin/asJava/KtLightClass;

12:44 PM	NoSuchMethodError: org.jetbrains.kotlin.asJava.LightClassUtilsKt.toLightClass(Lorg/jetbrains/kotlin/psi/KtClassOrObject;)Lorg/jetbrains/kotlin/asJava/KtLightClass;

12:44 PM	NoSuchMethodError: org.jetbrains.kotlin.asJava.LightClassUtilsKt.toLightClass(Lorg/jetbrains/kotlin/psi/KtClassOrObject;)Lorg/jetbrains/kotlin/asJava/KtLightClass;

12:44 PM	NoSuchMethodError: org.jetbrains.kotlin.asJava.LightClassUtilsKt.toLightClass(Lorg/jetbrains/kotlin/psi/KtClassOrObject;)Lorg/jetbrains/kotlin/asJava/KtLightClass;

12:44 PM	NoSuchMethodError: org.jetbrains.kotlin.asJava.LightClassUtilsKt.toLightClass(Lorg/jetbrains/kotlin/psi/KtClassOrObject;)Lorg/jetbrains/kotlin/asJava/KtLightClass;

12:44 PM	NoSuchMethodError: org.jetbrains.kotlin.asJava.LightClassUtilsKt.toLightClass(Lorg/jetbrains/kotlin/psi/KtClassOrObject;)Lorg/jetbrains/kotlin/asJava/KtLightClass;

12:44 PM	NoSuchMethodError: org.jetbrains.kotlin.asJava.LightClassUtilsKt.toLightClass(Lorg/jetbrains/kotlin/psi/KtClassOrObject;)Lorg/jetbrains/kotlin/asJava/KtLightClass;

12:44 PM	NoSuchMethodError: org.jetbrains.kotlin.asJava.LightClassUtilsKt.toLightClass(Lorg/jetbrains/kotlin/psi/KtClassOrObject;)Lorg/jetbrains/kotlin/asJava/KtLightClass;

12:44 PM	NoSuchMethodError: org.jetbrains.kotlin.asJava.LightClassUtilsKt.toLightClass(Lorg/jetbrains/kotlin/psi/KtClassOrObject;)Lorg/jetbrains/kotlin/asJava/KtLightClass;

12:45 PM	NoSuchMethodError: org.jetbrains.kotlin.asJava.LightClassUtilsKt.toLightClass(Lorg/jetbrains/kotlin/psi/KtClassOrObject;)Lorg/jetbrains/kotlin/asJava/KtLightClass;

12:45 PM	NoSuchMethodError: org.jetbrains.kotlin.asJava.LightClassUtilsKt.toLightClass(Lorg/jetbrains/kotlin/psi/KtClassOrObject;)Lorg/jetbrains/kotlin/asJava/KtLightClass;

12:45 PM	NoSuchMethodError: org.jetbrains.kotlin.asJava.LightClassUtilsKt.toLightClass(Lorg/jetbrains/kotlin/psi/KtClassOrObject;)Lorg/jetbrains/kotlin/asJava/KtLightClass;

12:45 PM	NoSuchMethodError: org.jetbrains.kotlin.asJava.LightClassUtilsKt.toLightClass(Lorg/jetbrains/kotlin/psi/KtClassOrObject;)Lorg/jetbrains/kotlin/asJava/KtLightClass;

12:45 PM	NoSuchMethodError: org.jetbrains.kotlin.asJava.LightClassUtilsKt.toLightClass(Lorg/jetbrains/kotlin/psi/KtClassOrObject;)Lorg/jetbrains/kotlin/asJava/KtLightClass;

12:45 PM	NoSuchMethodError: org.jetbrains.kotlin.asJava.LightClassUtilsKt.toLightClass(Lorg/jetbrains/kotlin/psi/KtClassOrObject;)Lorg/jetbrains/kotlin/asJava/KtLightClass;

12:45 PM	NoClassDefFoundError: org/jetbrains/kotlin/asJava/KtLightClassForExplicitDeclaration

12:45 PM	NoClassDefFoundError: org/jetbrains/kotlin/asJava/KtLightClassForExplicitDeclaration

12:45 PM	NoClassDefFoundError: org/jetbrains/kotlin/asJava/KtLightClassForExplicitDeclaration

12:46 PM	NoClassDefFoundError: org/jetbrains/kotlin/asJava/KtLightClassForExplicitDeclaration

```

**Comment by outlying** [comment]
Seems like the same issue with:
```
Spec Plugin 0.2.5
Kotlin 1.0.3
Android Studio 2.2
```

**Comment by raniejade** [comment]
@outlying are you able to upgrade the kotlin plugin to `1.0.4`?

**Comment by hannesstruss** [comment]
Seeing the same issue with Kotlin 1.1-M01-IJ2016.1-29

**Comment by raniejade** [comment]
Should be fix at 8f487112c6d3ee17321ebba7e4a8cbb0f355181c, please upgrade the kotlin plugin to `1.0.4`. @hannesstruss I've filed #14 for your issue.
